samples/document-snippets/instance.js

```diff
@@ -0,0 +1,382 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const Bigtable = require('@google-cloud/bigtable');
+const bigtableClient = new Bigtable();
+
+function newInstance(instanceId) {
+  // [START bigtable_create_instance]
+  const instance = bigtableClient.instance(instanceId);
+
+  // options for a PRODUCTION Instance
+  const options = {
+    clusters: [
+      {
+        name: 'sample-cluster',
+        nodes: 3,
+        location: 'us-central1-f',
+        storage: 'ssd',
+      },
+    ],
+    type: 'PRODUCTION', // Optional as default type is PRODUCTION
+  };
+
+  // // options for a DEVELOPMENT Instance
+  // const options = {
+  //   clusters: [
+  //     {
+  //       name: 'sample-cluster',
+  //       location: 'us-central1-f',
+  //       storage: 'hdd',
+  //     },
+  //   ],
+  //   type: 'DEVELOPMENT',
+  // };
+
+  // creates a new Instance
+  instance
+    .create(options)
+    .then(result => {
+      let newInstance = result[0];
+      // let operations = result[1];
+      // let apiResponse = result[2];
+
+      console.log(`Created Instance: ${newInstance.name}`);
+    })
+    .catch(err => {
+      console.error('Error creating prod-instance:', err);
+    });
+  // [END bigtable_create_instance]
+}
+
+function newCluster(clusterId, instanceId) {
+  // [START bigtable_create_cluster]
+  const instance = bigtableClient.instance(instanceId);
+
+  const options = {
+    location: 'us-central1-b',
+    nodes: 3,
+    storage: 'ssd',
+  };
+
+  instance
+    .createCluster(clusterId, options)
+    .then(result => {
+      const newCluster = result[0];
+      // const operations = result[1];
+      // const apiResponse = result[2];
+      console.log(`Cluster created: ${newCluster.name}`);
+    })
+    .catch(err => {
+      console.error('Error creating cluster: ', err);
+    });
+  // [END bigtable_create_cluster]
+}
+
+function newAppProfile(appProfileId, clusterId, instanceId) {
```

Comment:
Please call this `createAppProfile`.  Also, please reverse the order of parameters.  This should be `instanceId, clusterId, appProfileId`.